draft-ietf-quic-qlog-quic-events.md

```diff
@@ -323,21 +333,25 @@ ConnectivityConnectionClosed = {
     ? internal_code: uint32
     ? reason: text
     ? trigger:
-        "clean" /
-        "handshake_timeout" /
         "idle_timeout" /
-        ; this is called the "immediate close" in the QUIC RFC
+        "application" /
         "error" /
-        "stateless_reset" /
         "version_mismatch" /
-        ; for example HTTP/3's GOAWAY frame
-        "application"
+        ; when received from peer
+        "stateless_reset" /
+        ; when it is unclear what triggered the CONNECTION_CLOSE
+        "unspecified"
 
     * $$connectivity-connectionclosed-extension
 }
 ~~~
 {: #connectivity-connectionclosed-def title="ConnectivityConnectionClosed definition"}
 
+Loggers should use the most descriptive trigger for a `connection_closed` event
```

Comment:
Should or SHOULD

Reply:
Discussed in editors' meeting. Change to `SHOULD`

Reply:
```suggestion
Loggers SHOULD use the most descriptive trigger for a `connection_closed` event
```